frontend/src/components/HomePage.jsx

```diff
@@ -6,15 +6,19 @@ import NavBar from "./NavBar";
 import Intervention from "./Intervention";
 import { Switch, Route } from "react-router-dom";
 import Profil from "./Profil";
+import ContactUs from "../pages/ContactUs";
 
 export default function HomePage() {
   return (
     <div>
-      <CarCard />
       <Switch>
-        <Route path="/historic" component={Historic}></Route>
-        <Route exact path="/" component={ListCar}></Route>
-        <Route path="/intervention/:family" component={Intervention}></Route>
+        <Route path="/contact" component={ContactUs}></Route>
+        <CarCard />
+        <Switch>
```

Comment:
I might be wrong, but I don't think this is correct
We already discussed the idea to have several `<Switch>` in your app, and I can see the benefit (pages without header/footer, was it ?), but I really don't see what you're trying to accomplish here ?

Reply:
I don't know if that's a bad or a good practice..
but it's working like I want, maybe it should not ? 
what I want : 
the header is a car card with vehicle informations. useless for my page.
my page is a form to contact admin. 
and I want to keep the navbar, if I want to go back or anywhere I want to go without send message.

Reply:
maybe it's better to create my form the page login/signup withtout this switch.
and call under my form the component navbar?